File: FrontEnd/src/app/features/reservations/shipCrews/classes/dtos/shipCrew-write-dto.ts
export interface ShipCrewWriteDto {

    // PK
    id: number
    // FKs
    genderId: number
    nationalityId: number
    shipId: number
    specialtyId: number
    // Fields
    lastname: string
    firstname: string
    birthdate: string
    passportNo: string
    passportExpiryDate: string
    isActive: boolean
    // Rowversion
    putAt: string

}

File: API/Features/Reservations/ShipCrews/Dtos/ShipCrewWriteDto.cs
using API.Infrastructure.Interfaces;

namespace API.Features.Reservations.ShipCrews {

    public class ShipCrewWriteDto : IBaseEntity, IMetadata {

        // PK
        public int Id { get; set; }
        // FKs
        public int GenderId { get; set; }
        public int NationalityId { get; set; }
        public int ShipId { get; set; }
        // Fields
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public bool IsActive { get; set; }
        // Metadata
        public string PostAt { get; set; }
        public string PostUser { get; set; }
        public string PutAt { get; set; }
        public string PutUser { get; set; }

    }

}